tests/fuzz/wasm-mutator-fuzz/wasm_mutator_fuzz.cc

```diff
@@ -13,6 +13,149 @@
 
 using namespace std;
 
+static bool
+is_supported_val_kind(wasm_valkind_t kind)
+{
+    return kind == WASM_I32 || kind == WASM_I64 || kind == WASM_F32
+           || kind == WASM_F64 || kind == WASM_EXTERNREF
+           || kind == WASM_FUNCREF;
+}
+
+static wasm_val_t
+pre_defined_val(wasm_valkind_t kind)
+{
+    if (kind == WASM_I32) {
+        return wasm_val_t{ .kind = WASM_I32, .of = { .i32 = 2025 } };
```

Comment:
Would it be better to generate random value to cover more value range?

Reply:
😄 It makes it easier to replicate the issue that led to the failure. In a fuzzing issue report, you typically only get the error information generated by XSAN, with no accompanying logs. Using random values as function parameters can make it challenging to identify the exact combination that triggered the problem.

🤔 Additionally, I'm looking for a way to run opcodes with random operands. It appears to be a variant of the `LLVMFuzzerTestOneInput()` function. Perhaps I should create a new test `LLVMFuzzerTestOneInput()` to execute each opcode individually, allowing the fuzzer to supply random operands.